hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -995,18 +987,14 @@ def delete_atom(self, handle: str, **kwargs) -> None:
     def create_field_index(
         self,
         atom_type: str,
-        field: Optional[str] = None,
+        fields: List[str],
         type: Optional[str] = None,
         composite_type: Optional[List[Any]] = None,
-        fields: Optional[List[str]] = None,
         index_type: Optional[FieldIndexType] = None,
     ) -> str:
         if type and composite_type:
             raise ValueError("Both type and composite_type cannot be specified")
 
-        if field is None and fields is None:
-            raise ValueError("No field to create index: field and fields can not be None")
```

Comment:
Same here. Shouldn't `fields` be checked to ensure it's not empty?

Reply:
Added verification